install.sh

```diff
@@ -42,19 +46,24 @@ nvm_profile_is_bash_or_zsh() {
 # NVM_SOURCE always takes precedence unless the method is "script-nvm-exec"
 #
 nvm_source() {
+  local NVM_GITHUB_USER
+  NVM_GITHUB_USER="nvm-sh"
+  if [ -n "$NVM_INSTALL_GITHUB_USER" ]; then
+    NVM_GITHUB_USER="$NVM_INSTALL_GITHUB_USER"
+  fi
   local NVM_METHOD
   NVM_METHOD="$1"
   local NVM_SOURCE_URL
   NVM_SOURCE_URL="$NVM_SOURCE"
   if [ "_$NVM_METHOD" = "_script-nvm-exec" ]; then
-    NVM_SOURCE_URL="https://raw.githubusercontent.com/nvm-sh/nvm/$(nvm_latest_version)/nvm-exec"
+    NVM_SOURCE_URL="https://raw.githubusercontent.com/${NVM_GITHUB_USER}/nvm/$(nvm_latest_version)/nvm-exec"
```

Comment:
Please revert all the changes in this file.

Reply:
This would allow to install from another place than the main repo, I was not sure it could help, it was usefull on my side to check the installation from my fork.
But indeed if it is not wanted I will remove that.

Reply:
When developing, if your $NVM_DIR is a git repo, you can use git commands - changing the the install script would only be needed to test the install script, and the install script can only work on proper bash.

Reply:
The problem is that on Windows (GitBash) I cannot clone the repository due to some special characters in the name of some tests filename. That is why I needed to install it without the repository.
Anyway I managed to clone it with Cygwin and I made a symlink to point to that place so that is ok,